dlt/common/runtime/slack.py

```diff
@@ -0,0 +1,17 @@
+import requests
+from dlt.common import json, logger
+
+
+def send_slack_message(incoming_hook: str, message: str, is_markdown: bool = True) -> None:
+    """Sends a `message` to  Slack `incoming_hook`, by default formatted as markdown. Returns http status code"""
+    r = requests.post(incoming_hook,
+        data= json.dumps({
```

Comment:
There's an extra space between = and json. flake8 is likely not strict enough ;)